packages/lib/server/queries/teams/index.ts

```diff
@@ -92,6 +97,13 @@ export async function getTeamWithMembers(id?: number, slug?: string, userId?: nu
   const where: Prisma.TeamFindFirstArgs["where"] = {};
 
   if (userId) where.members = { some: { userId } };
+  if (orgSlug) {
+    where.parent = {
+      OR: [{ slug: orgSlug }, { metadata: { path: ["requestedSlug"], equals: orgSlug } }],
+    };
+  } else {
+    where.parentId = null;
```

Comment:
This is a bug fix, where it was not being explicitly set to null and was selecting any team org or not with the passed slug